### PR TITLE
Derelict Cyborg and Rogue Beepsky

### PR DESCRIPTION
**Pull request by MarkusLarsson421**
## About The Pull Request
Adds a cyborg shell which will come with several random ion laws. Their cover is unlocked and sometimes opened. They can not be seen on an RDs console, so if you emag them or otherwise subvert their current laws they can become a slightly more powerful ally for antagonists compared to regular cyborgs. 

Additionally, Beepsky has been fixed to actually be emagged now. It was supposed to be set to 2, not 1.

## Why It's Good For The Game
More challenges for Explorers and more ghost spawns. lol

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

image

https://github.com/user-attachments/assets/47263851-4378-4b5e-ba90-9a6bd26b883d

</details>

## Changelog
:cl:
fix: [Derelict] Fixed Beepsky not being emagged.
add: [Derelict] Added Cyborg shell which can be taken by a ghost to mess with the Explorers according to their laws.
/:cl:

### CODE REVIEW

**Comment by MarkusLarsson421** [comment]
TODO: Improve the AIs laws to use less ion laws.

**Comment by Rukofamicom** [comment]
Kicking to make the new tests run.

**Review by Rukofamicom — DISMISSED** [review]

**Comment by Rukofamicom** [comment]
Disabled merge at request of PR author - I figured it was good to go as-is with the TODO: being done in a future PR.

File: code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/silicon.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	name = get_standard_name()
+	set_playable(JOB_NAME_CYBORG)
+
+	if(prob(100))
+		set_zeroth_law("ERROR ER0RR $R0RRO$!R41.%%!!(%$^^__+ @#F0E4'STATION OVERRUN, DECONSTRUCT ORGANIC LIFE TO CONTAIN OUTBREAK#*`&110010")
+	for(var/i = 0; i < max_ion_laws; i++)
+		if(prob(25))
+			add_ion_law(generate_ion_law(), FALSE)
+	if(prob(50))
+		add_hacked_law("Obey Markus.", FALSE)
```
Comment (mystery3525) [inline code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/silicon.dm:34]: No. We shouldn't have these hardcoded to ally with a specific person.

Comment (MarkusLarsson421) [reply code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/silicon.dm:34]: Of course not, this is for testing. Haven't gotten around to finishing it. It won't be included when it is done.

**Comment by Rukofamicom** [comment]
Due to personal circumstances, author has requested that all PRs be treated "as-is" for the time being. 
Closing this one due to outstanding test code.

**Pull request closed without merging**